htdocs/delivery/class/delivery.class.php

```diff
@@ -1007,7 +1007,7 @@ public function getRemainingDelivered()
 	 *	@param      integer 		$delivery_date     Delivery date
 	 *	@return     int         						<0 if KO, >0 if OK
 	 */
-    public function set_delivery_date($user, $delivery_date)
+    public function setDeliveryDate($user, $delivery_date)
 	{
         // phpcs:enable
```

Comment:
here can delete the two phpcs disable enable

Reply:
Done, thanks Frederic.

Reply:
:+1: